src/handlers/completion.rs

```diff
@@ -179,12 +182,12 @@ async fn find_completions(
     })
 }
 
-fn new_string_arg_completion(value: String) -> CompletionItem {
+fn new_string_arg_completion(value: &String) -> CompletionItem {
     CompletionItem {
         deprecated: false,
         commit_characters: None,
         detail: None,
-        label: format!("\"{}\"", value),
+        label: format!("\"{}\"", value.trim_end()),
```

Comment:
why is the `trim_end` needed?

Reply:
extra line break returned from vscode, could also happen in other client i believe

Reply:
when would value have an extra line break?

Reply:
I can only saw that in measurements.

Reply:
but those measurements are coming from a function in VSCode that we wrote?

Reply:
I am not sure where does it come from.